packages/get-packages/README.md

```diff
@@ -0,0 +1,43 @@
+# Get Packages
+
+> A simple utility to get workspaces, whether they be yarn or bolt.
+
+This library exports a very simple function that looks at a package.json, and generates
+the glob of accepted workspaces from the `workspaces` field. It is intended mostly for
+use of developers building tools that want to support both kinds of mono-repos as an easy
+way to write tools for both.
+
+```javascript
+import getPackages from "@manypkg/get-packages";
+
+const { tool, root, packages } = await getPackages();
+```
+
+Workspaces have the shape:
+
+```typescript
+type Tool = "yarn" | "bolt" | "pnpm" | "root";
+
+export type Package = { packageJson: PackageJSON; dir: string };
+
+type Packages = {
+  tool: Tool;
+  packages: Package[];
+  root: Package;
+};
+
+export function getPackages(cwd: string): Promise<Packages>;
+export function getPackagesSync(cwd: string): Packages;
```

Comment:
This info seems not ideal as this is not a normal way of explaining imports.

This should really just be the import statements. Having the tool, package, and packages types is useful.

Speaking of, can we expose these as exports? Having access to types is nice.

Reply:
Do you want me to add exports to the types?